<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldSettings</class>
 <widget class="QWidget" name="WorldSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>World Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Fira Code SemiBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>World Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useProcSkyboxChkBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Use Procedural Skybox</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tonemap Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tonemapMode">
       <item>
        <property name="text">
         <string>ACES</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Filmic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lottes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reinhard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reinhard_V2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uchimura</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uncharted 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unreal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Anti-Aliasing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="aaMode">
       <item>
        <property name="text">
         <string>NoAA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FXAA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TAA</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scene Sampling pattern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sceneSamplingPattern">
       <item>
        <property name="text">
         <string>Halton</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stratified</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Fira Code Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Bloom Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="filterRadiusLbl">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Fira Code Light</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>filterRadius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="strengthLbl">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Fira Code Light</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>strength</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="filterRadius"/>
        </item>
        <item>
         <widget class="QLineEdit" name="strength"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>Fira Code Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Debug Mode Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="text">
         <string>Quad Overdraw</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>Unlit Shading</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>Show UV coords</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <family>Fira Code Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Renderer Features (Passes List)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Flags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>HiZ depth pre-pass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Shadows">
        <property name="text">
         <string>Shadows</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Skybox">
        <property name="text">
         <string>Skybox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SSAO">
        <property name="text">
         <string>SSAO</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ImGui">
        <property name="text">
         <string>ImGui</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="FXAA">
        <property name="text">
         <string>FXAA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="TAA">
        <property name="text">
         <string>TAA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Tonemapping">
        <property name="text">
         <string>Tonemapping</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>Skybox</zorder>
     <zorder>label_2</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>Shadows</zorder>
     <zorder>SSAO</zorder>
     <zorder>FXAA</zorder>
     <zorder>TAA</zorder>
     <zorder>Tonemapping</zorder>
     <zorder>ImGui</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
